### PR TITLE
Voter tutorial Starport v0.15.1 Add instructions for initializing vue

### PR DESCRIPTION
**Pull request by lilamila**
The [voter tutorial](https://tutorials.cosmos.network/voter/) requires Starport v0.15.1.

### CODE REVIEW

**Comment by barriebyron** [comment]
hi @lilamila thanks for your first-time contribution to making our tutorials more awesome. I don't know the technical substance behind your suggested changes, I am asking Denis... stay tuned!

**Comment by toschdev** [comment]
In version 0.15.1 the frontend should be running with `starport serve`. Visit localhost:1317
With v0.16 and higher, it needs to be executed manually.

**Comment by barriebyron** [comment]
@lilamila thanks for this PR, but no changes are required for the current Starport version required in this tutorial. You'll see this change come in at later Starport versions. Keep watching and letting us know when we can serve you better.

**Review by barriebyron — CHANGES_REQUESTED** [review]
this change is required only at Starport v0.16 and higher. Let's not make this change now.

File: voter/index.md [hunk 1]
```diff
@@ -143,9 +143,14 @@ With this command, you generated the code that handles the creation of `poll` it
...
 ### View the Front-end User Interface
 
-To see the front-end app form for creating polls:
+To see the front-end app form for creating polls navigate to the `vue` directory in a new tab/window:
+
+```sh
+cd vue
+npm install
+npm run serve
+```
```
[the remark this answers is not in the document]
Comment (fadeev) [reply voter/index.md:152]: v0.15 runs the front-end automatically.

**Pull request closed without merging**